main.py

```diff
@@ -12,6 +12,7 @@
 
 #Vars
 keyboard = Controller()
+SubtractDelay = 6 #Check ln 73
```

Comment:
This is not required the value can simply be put as a literal constant on line 73 since it's only used once.